face-kiosk/face-kiosk-application/src/main/java/com/pillar/skynet/facekiosk/application/Main.java

```diff
@@ -43,11 +50,21 @@ public static void main(String[] s) throws IOException, FrameGrabber.Exception {
 			LOG.info("Found a face tho.");
 		}
 
+		CvRect r = new CvRect(cvGetSeqElem(faces, 0));
+		cvSetImageROI(grayPhoto, r);
 
-//		OpenCVFaceRecognizer faceRecognizer = new OpenCVFaceRecognizer();
-//		//TODO: Need to resize picture to match training data before attempting to predict which face it is.
-//		faceRecognizer.train(Main.class.getResource("/faces/").getPath());
-//		String personName = faceRecognizer.predict());
-//		LOG.info("Hello " + personName);
+		IplImage cropped = cvCreateImage(cvGetSize(grayPhoto), grayPhoto.depth(), grayPhoto.nChannels());
+		cvCopy(grayPhoto, cropped);
+
+		IplImage resizedImage = IplImage.create(350, 425, cropped.depth(), cropped.nChannels());
```

Comment:
Captured Images have to be cropped to just the face and then resized to match the training data dimensions (here 350x425). Note that this is using the grayscale image, not the original. This is necessary for the eigenfischer face detection. Training data can be full color since it'll convert it for you.